layouts/shortcodes/docs/languages/index-intro.md

```diff
@@ -3,25 +3,26 @@
 <!-- prettier-ignore -->
 ` -}}
 {{ $lang := .Get 0 -}}
+{{ $translation := .Get 1 -}}
```

Comment:
Please accept the following change, and then revert all of the "en" argument additions to the `en` pages:

```suggestion
{{ $translation := .Get 1 | default "en" -}}
```